airflow/__init__.py

```diff
@@ -15,7 +15,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-#
+from __future__ import annotations
 
```

Comment:
Imports should _follow_ the docstring.

Reply:
Fixed. I am also reviewing all other PRs to apply it too.

Reply:
FYI -> I reviewed and updated all other "future annotations" PRs (actually some of them failed becaused of this problem).